packages/components/src/composite/legacy/test/index.tsx

```diff
@@ -47,7 +47,7 @@ async function renderAndValidate( ...args: Parameters< typeof render > ) {
 		const activeButton = queryByAttribute(
 			'data-active-item',
 			view.baseElement,
-			''
+			'true'
```

Comment:
With the new render function, could we actually remove the `renderAndValidate` utility in this file?

Reply:
Potentially!